src/total.jl

```diff
@@ -88,18 +114,30 @@ function total(x::Symbol, design::StratifiedSample)
     # the only difference between total and mean variance is the Nₕ instead of Wₕ
     V̂Ȳ̂ = sum((Nₕ .^ 2) .* (1 .- fₕ) .* s²ₕ ./ nₕ)
     SE = sqrt(V̂Ȳ̂)
-    return DataFrame(grand_total=grand_total, SE=SE)
+    return DataFrame(total=grand_total, SE=SE)
 end
 
 """
 Vectorise total operation over Vector{Symbol}
```

Comment:
Again, this docstring should be removed and combined with the corresponding functionality (starting at line 3). Also, the description is not corect. We are not vectorising the operation, we are implementing a method for it to accept a vector as input.

Reply:
“Vectorise total operation over Vector{Symbol}" isn't part of this PR and needs to be modified separately.

Reply:
If the docstring is removed it doesn't matter anyway.